### DISCUSSION TITLE
Image with full url in editor

### DISCUSSION THREAD

**Discussion by Kajaia** [austintoddj/canvas#1428] [General]
Is there any way to upload image in editor with full website url?

Now it uploads like this: `/storage/some/path/image.png`, and when using API this is how I get body:

```
"body": "<div class=\"embedded_image\" contenteditable=\"false\" data-layout=\"default\"><img alt=\"\" src=\"/storage/news/images/scEHNLZuNi5mhNTXUgzGiw2IsT5CeG41Q3sNsqdg.jpg\"></div><p>ProSoccer.TV is thrilled to welcome you to Football TV Match Schedules, your ultimate destination for up-to-date schedules of football matches from leagues around the globe. Whether you’re a die-hard fan of the Premier League, La Liga, Serie A, Bundesliga, Ligue 1, or any other top football league, we've got you covered. Plus, we provide comprehensive coverage of major tournaments such as the UEFA Champions League, Europa League, and the thrilling Euro Cup.</p>",
```

Maybe I can do it from config?

**Comment by austintoddj** [comment]
Closing with the intentional v7 media URL behavior ([upgrade guide](https://github.com/austintoddj/canvas/blob/main/.github/UPGRADE.md)):

- Public-disk library images and featured images are stored/served as **root-relative** paths (`/storage/...`), not absolute hosts derived from `APP_URL`.
- Unsplash / remote URLs stay absolute `https://…`.
- Non-local disks (S3, CDN) still return absolute object URLs.
- SEO, Open Graph, and webhooks expand local storage refs to absolute URLs when needed for external consumers.

If you need absolute URLs in your own API responses, expand them in the host app when you serialize the body.
